framework/tests/viewpointtest/TestgetAnnotatedLHS.java

```diff
@@ -0,0 +1,41 @@
+import viewpointtest.quals.A;
+import viewpointtest.quals.B;
+import viewpointtest.quals.ReceiverDependentQual;
+import viewpointtest.quals.Top;
+
+@ReceiverDependentQual
+class TestgetAnnotatedLHS {
```

Comment:
Rename file name and class name in this file.

Reply:
Can you rename this to `TestGetAnnotatedLhs`, to be consistent with the method name? Don't forget to also do the renaming in the comments elsewhere in this PR.

Reply:
Sorry I changed this locally but forgot to push it to the remote.